source/reference/limitations.txt

```diff
@@ -65,9 +64,7 @@ discuss your requirements.
 Unsupported Collection Types
```

Comment:
Now that there is only one unsupported collection type, consider making this heading more specific:

```suggestion
Time Series Collections
```

Reply:
Actually I see in PR #51 that we're adding another bullet to this section, so this can stay as-is.